runtime/src/iree/hal/drivers/webgpu/executable.c

```diff
@@ -111,7 +111,7 @@ static iree_status_t iree_hal_webgpu_create_wgsl_shader_module(
               .next = NULL,
               .sType = WGPUSType_ShaderModuleWGSLDescriptor,
           },
-      .code = code,
+      .source = code,
```

Comment:
Note that this sort of change breaks the native build, until `IREE_DAWN_TAG`, `third_party/webgpu_headers/`, etc. are updated to a compatible commit. How much do we care to support native and browser builds?